experimental/traffic-portal/src/app/core/users/users.component.html

```diff
@@ -11,9 +11,8 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<tp-header [title]="'Users'"></tp-header>
 <main>
-	<main>
+	<mat-card>
```

Comment:
This is why the e2e tests are failing; the selector for the page object is looking for `main > main` but you've changed that to `main > mat-card`. Which can probably just be `mat-card` since that's the only one on the page (whereas `main` wouldn't have worked because there were two of them).